imports/plugins/included/product-variant/register.js

```diff
@@ -48,8 +48,9 @@ Reaction.registerPackage({
     enabled: true,
     structure: {
       template: "products",
-      layoutHeader: "layoutHeader",
+      layoutHeader: "NavBar",
       layoutFooter: "",
+      // layoutFooter: "Footer", // TODO: decide if it's desirable to display standard footer in product grid for a stock installation
```

Comment:
No commented code.

Reply:
deleted comment. layout is retained as was, e.g. no footer will be displayed.